crates/router/src/core/payouts/helpers.rs

```diff
@@ -1109,6 +1111,8 @@ pub async fn update_payouts_and_payout_attempt(
             .or(payouts.payout_type.to_owned()),
         address_id: address_id.clone(),
         customer_id: customer_id.clone(),
+        sec_code,
+        account_type,
```

Comment:
minor nit - we can skip initializing the variables
```suggestion
        sec_code: req.sec_code,
        account_type: req.account_type,
```

Reply:
done